esmvalcore/preprocessor/_multimodel.py

```diff
@@ -433,6 +447,7 @@ def multi_model_statistics(products, span, statistics, output_products=None):
             statistics=statistics,
             output_products=output_products,
             span=span,
+            keep_originals=keep_originals,
```

Comment:
If this is never going to be exposed and only applies to `products`, why not fix it in the public function?

```suggestion
```python
    statistics_products= _multiproduct_statistics(
        products=products,
        statistics=statistics,
        output_products=output_products,
        span=span,
        )
    return output_products | products
```
```

Reply:
@stefsmeets when you suggest code changes via the Github interactive editor there is no need to add text formatting syntax (like ` ```python` here). Also @Peter9192 I would change `keep_originals` to something like `save_input_cubes` since originals refers more to the initial un-preprocced data and the option name doesn't tell the user what it does. Also note that for this to be in the documentation needs a mention of it :+1: